project/src/main/webapp/maps.js

```diff
@@ -0,0 +1,48 @@
+let map;
```

Comment:
It's not good practice to use global variables. I recommend using an ES6 class for this file and making these variables properties of the class. Check out [emailParse.js](https://github.com/Enoch-Chou/capstoneproject/blob/master/project/src/main/webapp/emailParse.js) for an example and ask Brandon, Enoch, or I if you have any questions!